### PR TITLE
chore: Move install-ses from Agoric SDK

### PR DESCRIPTION
**Pull request by kriskowal**
Refs #506

### CODE REVIEW

**Comment by kriskowal** [comment]
Appended three commits to migrate the name from `install-ses` to `init`.

File: packages/init/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@endo/init",
+  "version": "0.5.29",
+  "description": "Prepare Endo environment on import",
+  "type": "module",
+  "main": "index.js",
+  "exports": {
+    ".": "./index.js",
+    "./pre.js": "./pre.js",
```
Comment (michaelfig) [inline packages/init/package.json:9]: Needs:
```suggestion
    "./debug.js": "./debug.js",
    "./pre.js": "./pre.js",
```

File: packages/init/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// install-ses.js - call lockdown with default Agoric shims
```
Comment (michaelfig) [inline packages/init/index.js:1]:
```suggestion
// index.js - call lockdown with default Endo shims
```

**Review by michaelfig — APPROVED** [review]
Just some minor nits.  Feel free to merge after addressing.

**Pull request merged**